### PR TITLE
Feature/crontask

### PR DESCRIPTION
**Pull request by stasfly**
Cron tasks for mail sending

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -11,11 +11,14 @@ class User < ApplicationRecord
 
   # Include default devise modules. Others available are:
   # :confirmable,  :lockable, :timeoutable, :trackable and :omniauthable
+  scope :only_admins, -> { where(role: %i[super_admin admin]) }
```
Comment (olcplif) [inline app/models/user.rb:14]: redundant scope

File: app/views/test_notifier_mailer.html.egb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<body>
+  <h1> Test :-)) </h1>
+  <h3>
+    DID YOU KNOW?
+  </h3>
+  <p>
+    @@@@@@@@  Test  @@@@@@@@
+  </p>
+  <br>
+</body>
```
Comment (olcplif) [inline app/views/test_notifier_mailer.html.egb:10]: empty line

File: app/views/test_notifier_mailer.text.egb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+===============================================
+
+DID YOU KNOW?
+
+!!!!TEST!!!!
+
+user's email: <% @user.email %>
+
+===============================================
```
Comment (olcplif) [inline app/views/test_notifier_mailer.text.egb:9]: empty line

File: config/schedule.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+test_mailer_job:
+  cron: "* * * * *"
+  class: "TestMailerJob"
+  queue: default
```
Comment (olcplif) [inline config/schedule.yml:4]: empty line

File: config/sidekiq.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+development:
+  :concurrency: 1
+production:
+  :concurrency: 1
+:queues:
+  - default
```
Comment (olcplif) [inline config/sidekiq.yml:6]: empty line

File: db/schema.rb [hunk 6]
```diff
@@ -89,72 +89,21 @@
     t.index ["ancestry"], name: "index_documents_on_ancestry"
   end
```
Comment (olcplif) [inline db/schema.rb:91]: tables for lit disappear

File: coverage/.last_run.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (olcplif) [inline coverage/.last_run.json:1]: this folder is in .gitignore

**Review by olcplif — CHANGES_REQUESTED** [review]
please, fix these moments

**Review by mmrshk — CHANGES_REQUESTED** [review]
Why you have changes from another PR's???

**Pull request closed without merging**